src/controllers/controller.bar.js

```diff
@@ -358,8 +358,8 @@ module.exports = DatasetController.extend({
 			}
 		}
 
-		// No stacks? that means there is no visible data. Lets still initialize a `undefined`
-		// stack where possible invisible bars will be located.
+		// No stacks? that means there is no visible data. Let's still initialize an `undefined`
+		// stack, where possible invisible bars will be located.
```

Comment:
I think it made more sense before without this comma :-)

Reply:
True, fixed